pvlib/snow.py

```diff
@@ -13,30 +13,36 @@ def _time_delta_in_hours(times):
     return delta.dt.total_seconds().div(3600)
 
 
-def fully_covered_nrel(snowfall, threshold_snowfall=1.):
+def fully_covered_nrel(snowfall, snow_depth=None, threshold_snowfall=1.):
     '''
     Calculates the timesteps when the row's slant height is fully covered
     by snow.
 
     Parameters
     ----------
-    snowfall : Series
-        Accumulated snowfall in each time period [cm]
-
-    threshold_snowfall : float, default 1.0
+    snowfall: Series
+        Accumulated snowfall in each time period. [cm]
+    snow_depth: Series, optional
+        Snow depth on the ground at the beginning of each time period.
+        Must have the same index as `snowfall`. [cm]
+    threshold_snowfall: float, default 1.0
         Hourly snowfall above which snow coverage is set to the row's slant
         height. [cm/hr]
 
     Returns
     ----------
-    boolean: Series
+    Series
         True where the snowfall exceeds the defined threshold to fully cover
         the panel.
 
     Notes
     -----
     Implements the model described in [1]_ with minor improvements in [2]_.
 
+    `snow_depth` is used to set coverage=0 when no snow is present on the
+    ground. This check is described in [2]_ as needed for systems with
+    low tilt angle.
```

Comment:
I think this explanation fits better the `coverage_nrel` function, right? I guess here it should be something like:
    `snow_depth` is used to set snow_covered=False when no snow is present on the
    ground. This check is described in [2]_ as needed for systems with
    low tilt angle.